subsys/bluetooth/audio/aics.c

```diff
@@ -456,7 +456,8 @@ int bt_aics_register(struct bt_aics *aics, struct bt_aics_register_param *param)
 
 				chrc = aics->srv.service_p->attrs[i - 1].user_data;
 				attr->write = write_description;
-				attr->perm |= BT_GATT_PERM_WRITE_ENCRYPT;
+				attr->perm |= (BT_GATT_PERM_WRITE_ENCRYPT |
+							BT_GATT_PERM_WRITE_128B_ENTROPY);
```

Comment:
```suggestion
				attr->perm |= (BT_GATT_PERM_WRITE_ENCRYPT |
							BT_GATT_PERM_WRITE_128B_ENTROPY);
```
```suggestion
				attr->perm |= BT_GATT_PERM_WRITE_128B_ENTROPY;
```

Reply:
Yes, of course. Can't believe I missed this one xD